build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "com.android.tools.build:gradle:1.2.3"
+        classpath "com.jakewharton.sdkmanager:gradle-plugin:0.12.0"
+    }
+}
+
+apply plugin: "android-sdk-manager"
```

Comment:
Basically it will download SDK and all required SDK-related components such as required build tools and support library automatically. [Details](https://github.com/JakeWharton/sdk-manager-plugin/blob/master/README.md).